tutorials/getting-started-with-flow/02__listing-entities-in-a-grid/content.adoc

```diff
@@ -14,17 +14,19 @@ In this part of the tutorial we cover how to display data in a grid in your UI.
 Before getting into actual Vaadin development, we add the necessary domain objects and a dummy backend. In a real-world application, you would most probably have something similar implemented with JPA and EJB, or a Spring-based service.
 
 The first step is to copy three Java classes from GitHub and add them to your project. 
-We use these classes to build the UI. 
+We will use these classes to when building the UI. 
```

Comment:
Using present tense and active voice when possible is recommended for technical docs. I have also used it throughout. I recommend using the original.